src/pages/Auth/ForgotPasswordPage.jsx

```diff
@@ -1,7 +1,61 @@
+import {Link} from 'react-router-dom';
+
+import FormContainer from '../../components/FormContainer';
+
 function ForgotPasswordPage() {
     return (
         <>
-            <h4 className="w-full text-center">Forgot Password Page</h4>
+             {/* Construction Image */}
+             <div className="w-full lg:w-3/6 px-4 hidden md:block">
+                <img src={require("../../assets/images/construction.svg").default} alt="Construction"></img>
+            </div>
+
+            {/* Login Form */}
+            <div className="w-full lg:w-2/6">
+                <FormContainer>
+                    <form>
+                        <div className="form-group">
+                            <h1 className="uppercase font-bold text-xl text-center pb-3">
+                                Forgot Password?
+                            </h1>
+                            <p className="text-xs text-center pb-3">
+                                Please enter your registered email ID and company name to reset password?
+                            </p>
+                        </div>
+
+                        <div className="form-group">
+                            <label className="form-label"
+                                htmlFor="email">
+                                Email
+                            </label>
+                            <input type="email"
+                                id="email"
+                                className="form-control"
+                                placeholder="Email"
+                            />
+                        </div>
+
+                        <div className="form-group">
```

Comment:
Company name field is not required. Email alone is enough

Reply:
Ok